### PR TITLE
refactor(venues):Add unique constraints to venue names

### PR DESCRIPTION
**Pull request by beefysalad**
## Description
Add unique constraints on venue model

## Checklist

- [x] Reviewed own code
- [ ] Commented on code that is hard to understand
- [ ] Implemented tests for the feature/bugfix
- [ ] All **_Required_** GitHub status checks pass
- [ ] The frontend and backend PR previews have been deployed

**Backend only**

- [ ] Added test data for new entity
- [ ] Generated the client api if there are new or deleted endpoints and/or DTOs

### CODE REVIEW

**Review by junix033101 — DISMISSED** [review]
will take note on these changes and update my tests as well

**Review by noice-noise — DISMISSED** [review]
Thanks for implementing the changes! Please wait for Princh's approval before merging this PR.

**Review by princhcanal — DISMISSED** [review]
❤️

**Review by princhcanal — APPROVED** [review]

File: backend/prisma/migrations/20220912040324_unique_venues_name/migration.sql
Comment (beefysalad) [inline backend/prisma/migrations/20220912040324_unique_venues_name/migration.sql:8]: (c) @princhcanal

**Pull request merged**
